projects/026-play-bingo/python/main.py

```diff
@@ -0,0 +1,189 @@
+#         call = random.choice(numbers_list)
+#         calls_ingame += 1
+#         numbers_list.remove(call)
+#         cross_num(crossed_card, call)
+#         is_card_win = check_card(crossed_card)
+    
+#     copy_card = copy.deepcopy(my_bingo_card)
+#     calls_list.append(calls_ingame)    
+
+# minimum = min(calls_list)
+# maximum = max(calls_list)
+# avarage = sum(calls_list)/len(calls_list)
+
+# print(f'The minimum number of calls for the win is: {minimum}')
+# print(f'The maximum number of calls for the win is: {maximum}')
+# print(f'The avarage number of calls for the win is: {int(avarage)}')
+
+import random
+import copy
+
+def check_card_elements(card):
+    x = len(card[0])
+    y = len(card)
+    to_check = [
+        [[],[],[],[],[]],
+        [[],[],[],[],[]],
+        [[],[]]
+        ]
+    for row in range(x):
+        for column in range(y):
+            to_check[0][row].append(card[row][column])
+            to_check[1][row].append(card[column][row])
+           
+            if row == column:
+                to_check[2][0].append(card[column][row])
+                
+            if row + column == 4:
+                to_check[2][1].append(card[column][row])
+    for element in to_check:
+        for sub_element in element:
+            if sum(sub_element) == 0:
+                return True
+            
+    return False 
+
+def create_bingo_card():
+    bingo_card = []
+    step = 0
+    for letter in range(len('BINGO')):
+        bingo_card.append(random.sample(range(1+step, 15+step), k = 5))
+        step += 15
+    return bingo_card 
+
+def cross_card(card, call_number):
+    for line in range(len(card)):
+        for number in range(len(card[line])):
+            if card[line][number] == call_number:
+                card[line][number] = 0
+                return card
+    return card
+
+def display_card(card):
+    x = len(card[0])
+    y = len(card)
+    to_print = [
+        [[],[],[],[],[]],
+        [[],[],[],[],[]],
+        [[],[],[],[],[]],
+        [[],[],[],[],[]],
+        [[],[],[],[],[]],
+        ]
+    for column in range(x):
+        for row in range(y):
+            to_print[row][column].append(card[column][row])
+    
+    print(' B  I  N  G  O')
+    for line in to_print:
+        n1, n2, n3, n4, n5 = line
+        line_print = '{:2d} {} {} {} {}'.format(*n1, *n2, *n3, *n4, *n5) 
+        print(line_print) 
+
+generated_card = create_bingo_card()
+bingo_calls = random.sample(range(1,76), k=75)
+calls_list = []
+
+for games in range(0,1000):
+    playing_card = copy.deepcopy(generated_card)
+    random.shuffle(bingo_calls)
+    for numbers in range(len(bingo_calls)):
+        n_call = bingo_calls[numbers]
+        cross = cross_card(playing_card, n_call)
+        check = check_card_elements(cross)
+        if check:
+            calls_list.append(numbers+1)
+            break
+
+minimum = min(calls_list)
+maximum = max(calls_list)
+avarage = sum(calls_list)/len(calls_list)
```

Comment:
if you want a float number as you used the "/" operator instead "//", why you do a cast like this `int(avarage)` in` the `print` statement at line 188?
P.S. you can format the output without change the variable value. Basically is discouraged to use casting caused by performance issues